realsense_camera/README.md

```diff
@@ -4,24 +4,47 @@
 | Version          | Best Known (indigo)  | Best Known (kinetic) |
 |:---------------- |:---------------------|:---------------------|
 | Operating System | Ubuntu 14.04.4 LTS   | Ubuntu 16.04 LTS     |
-| Kernel           | 4.4.0-040400-generic | 4.4.0-22-generic     |
+| Kernel           | 4.4.0-34-generic     | 4.4.0-34-generic     |
 | Backend          | video4linux          | video4linux          |
 | librealsense     | 0.9.2                | 0.9.2                |
 | R200 Firmware    | 1.0.72.06            | 1.0.72.06            |
 | F200 Firmware    | 2.60.0.0             | 2.60.0.0             |
 | SR300 Firmware   | 3.10.10.0            | 3.10.10.0            |
 
 ###Installation:
-#####Getting the camera to work on Linux
+####Getting the camera to work on Linux
```

Comment:
Can we have this heading as "Installing the package" so its alternate option is "Building the package from source"